NumberBaseball/NumberBaseball/main.swift

```diff
@@ -1,10 +1,64 @@
 //
 //  NumberBaseball - main.swift
-//  Created by yagom. 
+//  Created by 애종, LJ. 
```

Comment:
```suggestion
//  Created by 애종, LJ. 
```
사소한 것도 고쳐주시면 좋을 것 같아요

Reply:
수정했습니다.😀